lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -831,6 +831,7 @@ namespace Js
         if (cse->ei.bstrDescription)
         {
             value = JavascriptString::NewCopySz(cse->ei.bstrDescription, scriptContext);
+            JavascriptOperators::OP_SetProperty(error, PropertyIds::description, value, scriptContext);
```

Comment:
>description [](start = 68, length = 11)

Isn't description IE specific?  where ever we use description instead of message seems like a compat issue?

Reply:
Yes this to fix the Edge/Jshost problem. In the Edge case we never show what kind of syntax error found in the module case. With the 'description' being populated we are showing that now. ( I was planning to add jshost test as well - with some fix on the full side)